src/sentry/integrations/github/actions/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .create_ticket import GitHubCreateTicketAction
+
+__all__ = ("GitHubCreateTicketAction",)
```

Comment:
How was this not already here if we have GH issue creation from rules?

Reply:
I think I forgot to do this in the original github pr. I believe it's only used for importing